sdk/synapse/mgmt-v2019_06_01_preview/src/main/java/com/microsoft/azure/management/synapse/v2019_06_01_preview/implementation/SensitivityLabelsImpl.java

```diff
@@ -67,7 +67,7 @@ public SynapseManager manager() {
     @Override
     protected Observable<SensitivityLabelInner> getInnerAsync() {
         SqlPoolSensitivityLabelsInner client = this.manager().inner().sqlPoolSensitivityLabels();
-        return client.getAsync(this.resourceGroupName, this.workspaceName, this.sqlPoolName, this.schemaName, this.tableName, this.columnName, SensitivityLabelSource.RECOMMENDED);
+        return client.getAsync(this.resourceGroupName, this.workspaceName, this.sqlPoolName, this.schemaName, this.tableName, this.columnName, SensitivityLabelSource.CURRENT);
```

Comment:
due to the PUT is `current` https://github.com/azure/azure-rest-api-specs/blob/216e7ae3aafefd2ea16a5b64a49281d21da6cb1a/specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json#L1745

Reply:
changed, parsed from id